src/containers/ImportResources/ImportResources.js

```diff
@@ -100,9 +119,18 @@ export class ImportResources extends Component {
     const gitresourcename = 'git-source';
     const gitcommit = 'master';
 
-    if (repourl === '' || !namespace) {
+    const valid = validateURL(repourl);
```

Comment:
Comments would be great here, had to focus to understand this more than I'd have liked to

Reply:
Sorry, my bad, I'll add a comment in